tv_apps/smart-clock/manifest.webapp

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Smart Clock",
+  "type": "certified",
+  "role": "widget",
+  "launch_path": "/widget.html",
+  "widgets": {
```

Comment:
The `widgetPages` array is required. Please find more information at https://wiki.mozilla.org/WebAPI/WidgetAPI . An example can be found at https://github.com/huchengtw-moz/gaia/tree/stingray-widget-api-v2 .